app/code/Magento/InventorySales/Plugin/InventoryApi/StockRepository/SaveSalesChannelsLinksPlugin.php

```diff
@@ -68,4 +116,49 @@ public function afterSave(
 
         return $stockId;
     }
+
+    /**
+     * @param StockInterface $stock
+     * @return array
+     */
+    private function getUnassignedWebsiteCodesForStock(StockInterface $stock): array
+    {
+        $assignedWebsiteCodes = $newWebsiteCodes = [];
+        $assignedSalesChannels = $this->getAssignedSalesChannelsForStock->execute((int)$stock->getStockId());
+
+        foreach ($assignedSalesChannels as $salesChannel) {
+            if ($salesChannel->getType() === SalesChannel::TYPE_WEBSITE) {
+                $assignedWebsiteCodes[] = $salesChannel->getCode();
+            }
+        }
+
+        $extensionAttributes = $stock->getExtensionAttributes();
+        $newSalesChannels = $extensionAttributes->getSalesChannels() ?: [];
+
+        foreach ($newSalesChannels as $salesChannel) {
+            if ($salesChannel->getType() === SalesChannel::TYPE_WEBSITE) {
+                $newWebsiteCodes[] = $salesChannel->getCode();
+            }
+        }
+
+        return array_diff($assignedWebsiteCodes, $newWebsiteCodes);
+    }
+
+    /**
+     * Return website codes assigned to default stock
+     *
+     * @return array
+     */
+    private function getAssignedToDefaultWebsiteCodes(): array
+    {
+        $result = [];
+        $assignedToDefaultStockSalesChannels = $this->getAssignedSalesChannelsForStock
+            ->execute($this->defaultStockProvider->getId());
+
+        foreach ($assignedToDefaultStockSalesChannels as $salesChannel) {
```

Comment:
The same... Why do we need to operate with website code?

Reply:
For save website codes assigned to default stock.